src/inputs/TextFieldBase.tsx

```diff
@@ -113,13 +111,15 @@ export function TextFieldBase<X extends Only<TextFieldXss, X>>(props: TextFieldB
     : [Palette.White, Palette.Gray100, Palette.Gray100];
 
   const fieldStyles = {
-    container: Css.df.fdc.w100.maxw(px(550)).relative.$,
+    container: Css.df.fdc.w100.maxw(px(550)).relative.if(labelStyle === "left").fdr.gap2.jcsb.aic.$,
     inputWrapper: {
       ...Css[typeScale].df.aic.br4.px1.w100
         .hPx(fieldHeight - maybeSmaller)
         .if(compact)
         .hPx(compactFieldHeight - maybeSmaller).$,
-      ...Css.bgColor(bgColor).gray900.if(contrast).white.$,
+      ...Css.bgColor(bgColor)
+        .gray900.if(contrast)
+        .white.if(labelStyle === "left").maxw50.$,
```

Comment:
Question: @roberthdg hm, does this `maxw50` handle making all of the labels line up?

Like I'm wondering what happens if we have:

```
| Some really long label   <field> |
| Short label <field>              |
```

Will the "really long label" vs. "really short label" mean that the two `<fields>` are no longer show vertically aligned?

I kind of assume not, which if so is a miss on my part, not thinking of that up-front / before starting this ticket, that to ensure the right-side fields are 100% vertically aligned, we probably need some sort of cross-`TextFieldBase` containers/constraints.

But, I dunno, I'm kind of guessing; @roberthdg  how does vertical alignment work out in the current approach?

Reply:
That `maxw50` would be for the `inputWrapper` (by default it expands to whatever width its available), it could also just be set to w50 to override the default w100.